packages/bigquery/src/index.js

```diff
@@ -66,7 +66,7 @@ function BigQuery(options) {
   }
 
   var config = {
-    baseUrl: 'https://www.googleapis.com/bigquery/v2',
+    baseUrl: 'https://www.googleapis.com./bigquery/v2',
```

Comment:
Just for the record, since this is a Google repo:

```
curl -I www.google.com.
HTTP/1.1 200 OK
Date: Wed, 12 Apr 2017 18:35:27 GMT
Expires: -1
Cache-Control: private, max-age=0
Content-Type: text/html; charset=ISO-8859-1
P3P: CP="This is not a P3P policy! See https://www.google.com/support/accounts/answer/151657?hl=en for more info."
Server: gws
X-XSS-Protection: 1; mode=block
X-Frame-Options: SAMEORIGIN
Set-Cookie: NID=101=j6R0GIlDZYi8xxjty0Zd3MBI7vcOJh8TKY64j9ah_JhkQKi7lNQc7j3N_PMK4uSrt1o7fNGLc32wrglaTZpxCYaQa17t86_LyOrLei2V_yg4rEFd8_0az9FtLCKAYu1zYNyBbzsJCJYhWsrnyQ; expires=Thu, 12-Oct-2017 18:35:27 GMT; path=/; domain=.google.com; HttpOnly
Transfer-Encoding: chunked
Accept-Ranges: none
Vary: Accept-Encoding
```

At least someone is doing it right ;)